imports/config/accounts.js

```diff
@@ -69,7 +81,19 @@ if (Meteor.isServer) {
     AccountsTemplates.configure({
         forbidClientAccountCreation: (Meteor.settings.public.forbidClientAccountCreation
                                         ? Meteor.settings.public.forbidClientAccountCreation
-                                        : false)
+                                        : false),
+
+        sendVerificationEmail: (Meteor.settings.public.sendVerificationEmail
+            ? Meteor.settings.public.sendVerificationEmail
+            : false),
+
+        showResendVerificationEmailLink: (Meteor.settings.public.showResendVerificationEmailLink
+            ? Meteor.settings.public.showResendVerificationEmailLink
+            : false),
+
+        showForgotPasswordLink: (Meteor.settings.public.showForgotPasswordLink
+            ? Meteor.settings.public.showForgotPasswordLink
+            : false)
```

Comment:
These three settings should only evalute to "true", if also the global setting `email.enableMailDelivery === true`